Tests/test_file_avif.py

```diff
@@ -146,7 +146,7 @@ def test_write_rgb(self, tmp_path: Path) -> None:
 
             # avifdec hopper.avif avif/hopper_avif_write.png
             assert_image_similar_tofile(
-                reloaded, "Tests/images/avif/hopper_avif_write.png", 6.02
+                reloaded, "Tests/images/avif/hopper_avif_write.png", 6.14
```

Comment:
Why do we need to relax the average pixel difference threshold?

Reply:
I changed this temporarily; it has since been changed back. It seems that when aom is built with `-Os` for manylinux2014 wheels (i.e. CentOS 7 with gcc 10) the encoder behavior changes. I think this must be a bug with gcc? I can't see why compiler optimization flags should affect the actual behavior of an application. I've opted to use `-DCMAKE_BUILD_TYPE=Release` for these el7 builds.